modules/xfem/src/base/XFEM.C

```diff
@@ -284,6 +284,9 @@ XFEM::update(Real time, NonlinearSystemBase & nl, AuxiliarySystem & aux)
     _mesh->skip_partitioning(true);
     _mesh->prepare_for_use();
 
+    //    _mesh->prepare_for_use(true,true); //doing this preserves the numbering, but generates
```

Comment:
There's a bunch of commented-out code throughout this class that was presumably for debugging. Please clean that up.

Reply:
this seems like an important comment to keep. It explains why the true, true parameters are not chosen above